src/theme/components/PageHeader.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2021 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import Headroom from 'react-headroom';
+import {
+  Box,
+  Link as RebassLink,
+  Flex,
+  Image,
+  Button,
+} from 'rebass/styled-components';
+import styled from 'styled-components';
+import { header } from '../../content/HeaderContent';
+import { PAGE, SECTION } from '../utils/constants';
+import Link from './Link';
+import { capitalize } from '../utils/string';
+
+const PageHeader = (): JSX.Element => {
```

Comment:
Almost duplicated from the Header class, but we should replace the Header in the home page & have the same in all pages later.

Reply:
Ok to me, we will see later when we will decide how we manage the header on all pages